src/pool.rs

```diff
@@ -468,13 +511,22 @@ impl<Conn: Connection> PoolInner<Conn> {
             break;
         }
 
+        #[cfg(feature = "probes")]
+        match &result {
+            Ok(handle) => {
+                probes::pool__claim__done!(|| (self.name.as_str(), id.0, handle.slot_id().as_u64()))
+            }
+            Err(_) => probes::pool__claim__failed!(|| (self.name.as_str(), id.0)),
```

Comment:
should we put the error message in here?

Reply:
As implemented, it'll always be "no backends", so I left it out. However, `slot__set__claim__failed` should be able to provide per-backend error messages, if they're being contacted.